README.md

```diff
@@ -15,12 +15,12 @@ Open Drug Discovery Toolkit (ODDT) is modular and comprehensive toolkit for use
   * Issues: https://github.com/oddt/oddt/issues
 
 ## Requrements
-  * Python 2.7 or 3.5+
-  * OpenBabel (2.3.2+) or/and RDKit (2016.03)
-  * Numpy (1.11+)
-  * Scipy (0.17+)
+  * Python 3.6+
+  * OpenBabel (3.0+) or/and RDKit (2018.03+)
+  * Numpy (1.12+)
+  * Scipy (0.19+)
   * Sklearn (0.18+)
```

Comment:
I can also see some `if sklearn_version >= '0.19'`, maybe we should get rid 0.18?